app/api/models/errors/TestSupportErrors.scala

```diff
@@ -24,18 +24,30 @@ object TypeOfBusinessFormatError      extends MtdError("FORMAT_TYPE_OF_BUSINESS"
 object AccountingTypeFormatError      extends MtdError("FORMAT_ACCOUNTING_TYPE", "The provided accountingType field is invalid", BAD_REQUEST)
 
 object RuleCommencementDateNotSupported
-    extends MtdError("RULE_COMMENCEMENT_DATE_NOT_SUPPORTED", "The specified commencementDate must be in the past", BAD_REQUEST)
+  extends MtdError("RULE_COMMENCEMENT_DATE_NOT_SUPPORTED", "The specified commencementDate must be in the past", BAD_REQUEST)
 
 object PostcodeFormatError         extends MtdError("FORMAT_POSTCODE", "The provided businessAddressPostcode is invalid", BAD_REQUEST)
 object LatencyIndicatorFormatError extends MtdError("FORMAT_LATENCY_INDICATOR", "The format of a latency indicator field is incorrect", BAD_REQUEST)
 object MissingPostcodeError        extends MtdError("MISSING_POSTCODE", "Missing postcode", BAD_REQUEST)
 object CountryCodeFormatError      extends MtdError("FORMAT_COUNTRY_CODE", "The provided country code is invalid", BAD_REQUEST)
 
 object RuleFirstAccountingDateRangeInvalid
-    extends MtdError("RULE_FIRST_ACCOUNTING_DATE_RANGE_INVALID", "The first accounting period is not a single complete tax year", BAD_REQUEST)
+  extends MtdError("RULE_FIRST_ACCOUNTING_DATE_RANGE_INVALID", "The first accounting period is not a single complete tax year", BAD_REQUEST)
 
 object MissingFirstAccountintPeriodStartDateError
-    extends MtdError("MISSING_FIRST_ACCOUNTING_PERIOD_START_DATE", "Missing firstAccountingPeriodStartDate value", BAD_REQUEST)
+  extends MtdError("MISSING_FIRST_ACCOUNTING_PERIOD_START_DATE", "Missing firstAccountingPeriodStartDate value", BAD_REQUEST)
 
 object MissingFirstAccountintPeriodEndDateError
-    extends MtdError("MISSING_FIRST_ACCOUNTING_PERIOD_END_DATE", "Missing firstAccountingPeriodEndDate value", BAD_REQUEST)
+  extends MtdError("MISSING_FIRST_ACCOUNTING_PERIOD_END_DATE", "Missing firstAccountingPeriodEndDate value", BAD_REQUEST)
```

Comment:
I appreciate you didn't change the names of these errors, but there is a typo in them! Could you change this please.

Reply:
Good spot. I didn't notice the typos.

Reply:
I think I fixed them now. Please double check.

https://github.com/hmrc/mtd-sa-test-support-api/commit/4766ddcc7bb067ec929be5a1ed72601f03820827